### ISSUE TITLE
Make tool pip-installable

### ISSUE THREAD

**Issue by corneliusroemer** [lenaschimmel/sc2rf#3]
Shouldn't be difficult, you need a setup.py and account of Pypi

You can have a look at this repo of mine that can be installed via Pypi as a command line tool (if you install it, the command becomes automatically available in Path!)
https://github.com/corneliusroemer/fasta_zstd_sqlite/blob/master/setup.py

**Comment by lenaschimmel** [comment]
That sounds like a great idea! I will probably do it soon(ish).

I think I want to read a bit about distutils and setuptools first so that I understand what I'm actually doing.

**Comment by corneliusroemer** [comment]
This would be so useful! It's really not hard ;)

**Comment by lenaschimmel** [comment]
I just added a `setup.py` but then I realized that sc2rf needs some data files to run, which is [possible](https://setuptools.pypa.io/en/latest/userguide/datafiles.html).


It may also modify one of those data files at runtime (`virus_properties.json`), [which is a problem](https://setuptools.pypa.io/en/latest/userguide/datafiles.html#accessing-data-files-at-runtime). So it's not as trivial as I hoped it would be. I think I have to leave it in this half-baked state for a short while…
